wwwroot/about.html

```diff
@@ -45,9 +45,6 @@
                     <li class="nav-item">
                         <a class="nav-link" href="index.html">The Wallet</a>
                     </li>
-                    <li class="nav-item">
-                        <a class="nav-link" href="contribution.html">Contribution</a>
-                    </li>
```

Comment:
Why this change?

Reply:
No space for more buttons at the header. Contribution moved to the footer.